pkg/backend/diy/backend.go

```diff
@@ -1059,6 +1060,9 @@ func (b *diyBackend) apply(
 	op backend.UpdateOperation, opts backend.ApplierOptions,
 	events chan<- engine.Event,
 ) (*deploy.Plan, sdkDisplay.ResourceChanges, error) {
+	resetKeepRunning := nosleep.KeepRunning()
+	defer resetKeepRunning()
```

Comment:
Chatted in Slack. We can't just 

```
defer nosleep.KeepRunning()
```

as executing `KeepRunning()` is what calls the API for inhibiting the sleep, so it needs to run first, and the function we return resets that, so the computer can go to sleep again.

> That said, does non-sleep matter in a DIY case? We don't have tokens whose expiry we are worried about, right?

I do think it's still helpful.  It's not as crucial because we don't have to worry about token expiry, but it's still not great if the computer goes to sleep while a pulumi up happens, as resource setup etc. might fail, so it's still better to have this than not I think.